glean-core/android/src/main/java/mozilla/telemetry/glean/Dispatchers.kt

```diff
@@ -177,8 +177,13 @@ internal object Dispatchers {
         /**
          * Helper function to execute the task as either an synchronous or asynchronous operation,
          * depending on whether [testingMode] is true.
+         *
+         * WARNING: THIS SHOULD ALMOST NEVER BE USED. IF IN DOUBT, USE [launch] INSTEAD.
```

Comment:
Can we document here why that is? What makes it so dangerous?

Reply:
Sure.  It shouldn't be used for anything that might be called before initialization that needs to actually happen after initialization.  I'll add that to the comment here.